Titanium.Web.Proxy/ProxyServer.cs

```diff
@@ -298,7 +298,11 @@ public Action<Exception> ExceptionFunc
         /// <summary>
         /// List of supported Ssl versions
         /// </summary>
+#if NET45

```

Comment:
Ssl3 is deprecated, it does not work with .NET Core. So maybe it sohuld be removed from the .NET Framework version, too. What do you think?